db/migrate/20160129081441_create_versions.rb

```diff
@@ -0,0 +1,20 @@
+class CreateVersions < ActiveRecord::Migration
+
+  # The largest text column available in all supported RDBMS is
+  # 1024^3 - 1 bytes, roughly one gibibyte.  We specify a size
+  # so that MySQL will use `longtext` instead of `text`.  Otherwise,
+  # when serializing very large objects, `text` might not be big enough.
+  TEXT_BYTES = 1_073_741_823
+
+  def change
+    create_table :versions do |t|
```

Comment:
If "versions" is a table for history of all types of objects, then it's probably appropriate as is... but if it is a separate table per object type, then yeah, it would be nice to have it name scoped to the type it is the history for.